apps/src/lib/util/AnalyticsReporter.js

```diff
@@ -12,13 +12,17 @@ import {
   isStagingEnvironment,
   isDevelopmentEnvironment,
 } from '../../utils';
+import statsigReporter from '@cdo/apps/lib/util/StatsigReporter';
 
 import {EVENT_GROUPS} from './AnalyticsConstants';
 
 import DCDO from '@cdo/apps/dcdo';
 
 // A flag that can be toggled to send events regardless of environment
 const ALWAYS_SEND = false;
+const AMPLITUDE = 'Amplitude';
+const STATSIG = 'Statsig';
+const BOTH = 'Both';
```

Comment:
Could be done in a follow-up but should these be accessible as constants from other places? So that the third param could be set to `ANALYTICS_REPORTER_CONSTANTS::AMPLITUDE` (or something)

Reply:
Yuuup good call. I moved them into the AnalyticsConstants file and imported them instead!